docs/docs/guides/labs/dg/creating-a-project.md

```diff
@@ -46,16 +46,16 @@ To use `tree`, install it with `brew install tree` (Mac), or follow the [install
 
 :::
 
-- The Python package `my-project` lives in `src/my-project` and contains the deployable code that defines
+- The Python package `my_project` lives in `src/my_project` and contains the deployable code that defines
```

Comment:
The quick start used `quick-start` as the project name. Would it be better for that to be `quick_start`

Reply:
I think that one can stay the same -- in this case, `my-project` is the project name, but `my_project` is the Python package; the directory structure looks like:

```
my-project
├── pyproject.toml
├── src
│   └── my_project
│       ├── __init__.py
│       ├── __pycache__
│       │   └── __init__.cpython-312.pyc
│       ├── components
│       │   ├── __init__.py
│       │   ├── __pycache__
│       │   │   └── __init__.cpython-312.pyc
│       │   └── shell_command.py
│       ├── definitions.py
│       └── defs
│           └── __init__.py
├── tests
│   └── __init__.py
└── uv.lock
```